cytominer_eval/utils/transform_utils.py

```diff
@@ -73,30 +53,30 @@ def convert_pandas_dtypes(df: pd.DataFrame, col_fix: type = np.float64) -> pd.Da
     return df
 
 
-def assert_pandas_dtypes(df: pd.DataFrame, col_fix: type = np.float64) -> pd.DataFrame:
+def assert_pandas_dtypes(df: pd.DataFrame, col_fix: type = float) -> pd.DataFrame:
     r"""Helper funtion to ensure pandas columns have compatible columns
 
     Parameters
     ----------
     df : pandas.DataFrame
         A pandas dataframe to convert columns
-    col_fix : {np.float64, np.str}, optional
+    col_fix : {float, str}, optional
         A column type to convert the input dataframe.
 
     Returns
     -------
     pd.DataFrame
         A dataframe with converted columns
     """
-    assert col_fix in [np.str, np.float64], "Only np.str and np.float64 are supported"
+    assert col_fix in [str, float], "Only str and float are supported"
 
     df = convert_pandas_dtypes(df=df, col_fix=col_fix)
 
     assert_error = "Columns not successfully updated, is the dataframe consistent?"
     if col_fix == np.str:
```

Comment:
Did you miss this one? 
```suggestion
    if col_fix == str:
```

Reply:
awesome, yes. Thanks!